apps/prairielearn/src/components/PersonalNotesPanel.html.ts

```diff
@@ -30,43 +30,36 @@ export function PersonalNotesPanel({
         <i class="fas fa-paperclip"></i>
         Personal Notes
       </div>
-      <table class="table table-sm">
-        <tbody>
-          ${fileList.length === 0
-            ? html`
-                <tr>
-                  <td><i>No attached notes</i></td>
-                </tr>
-              `
-            : fileList.map(
+      ${fileList.length === 0
```

Comment:
I rewrote this as an unordered list, which seems semantically more correct than a table.